src/EditorFeatures/Core/Extensibility/ExportVersionSpecificAttribute.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Editor
+{
+    internal enum VisualStudioVersion
+    {
+        VS2015 = 14,
```

Comment:
Note that these need to be ascending since you compare their raw numeric values later.